solana/program_v2/packages/tests/src/pass-suite/verify.test.ts

```diff
@@ -64,7 +70,7 @@ describe('Verify a pass', () => {
         gatekeeperAuthority.publicKey,
         mintAccount.publicKey,
         gatekeeperPDA,
-        3000
+        10000
```

Comment:
Is this change required?

Reply:
Changed for testing, forgot to switch back. Thanks for catching